docs/conf.py

```diff
@@ -31,6 +31,7 @@
 extensions = [
     "myst_nb",
     "sphinx_design",
+    "sphinx_copybutton",
```

Comment:
This extension adds a button to every code block so it's instantly copied to the readers clipboard when clicked.